src/migrations/20241002000000-clean-training-report-data.js

```diff
@@ -0,0 +1,19 @@
+const { prepMigration } = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      return queryInterface.sequelize.query(`
+         UPDATE "SessionReportPilots" SET data = data - 'event';
+         UPDATE "EventReportPilots" SET data = data - 'sessions';
```

Comment:
looks like these are keyed as sessionReports in the data not sessions so this line doesn't actually do anything

Reply:
Some of the sessions have event data, and the events have data from the sessions. They clean out the unneeded data from both.

Reply:
No. I mean: when I looked at the actual data I was seeing event.data.sessionReports not event.data.sessions.

Reply:
I dont know how I missed that